frontend/src/components/Features.js

```diff
@@ -0,0 +1,17 @@
+import '../styles/Features.css';
+
```

Comment:
I dont see React imported here 🤔

Reply:
Oh yea! That's weird cause it worked without one.